lizard_auth_server/tests/test_models.py

```diff
@@ -229,20 +229,20 @@ class StrMethodTestCase(TestCase):
 
     def test_str_portal(self):
         portal = factories.PortalF()
-        self.assertEquals(type(portal.__str__()), str)
+        self.assertEquals(type(str(portal)), str)
```

Comment:
In hoeverre is een echte smoke test hier handiger/duidelijker? `self.assertTrue(str(portal))` ?